package.json

```diff
@@ -106,7 +106,7 @@
     "@oclif/config": "^1.14.0",
     "@oclif/dev-cli": "1.22.0",
     "@oclif/parser": "^3.8.4",
-    "@octokit/plugin-throttling": "^2.7.1",
+    "@octokit/plugin-throttling": "^3.3.0",
```

Comment:
Required to get types for the module, changelog: https://github.com/octokit/plugin-throttling.js/releases/tag/v3.0.0
Basically drops node 8 and a slight change to the export which I updated in the one place we use it (typescript was able to catch the need to update it as well :) )